src/chrome/content/rules/Ruhr-Uni-Bochum.de.xml

```diff
@@ -62,7 +57,6 @@ Fetch error: http://www.rz.ruhr-uni-bochum.de/ => https://www.rz.ruhr-uni-bochum
 	<target host="res.mobsec.ruhr-uni-bochum.de" />
 	<target host="helpdesk.rz.ruhr-uni-bochum.de" />
 	<target host="linux.rz.ruhr-uni-bochum.de" />
-	<target host="www.rz.ruhr-uni-bochum.de" />
```

Comment:
Could you please add the host to the list of `Problematic domains`? The issue on this host in an incomplete cert chain.